### DISCUSSION TITLE
Customize Tree View of Collection in CP

### DISCUSSION THREAD

**Discussion by JimSeven** [statamic/cms#4523] [Q&A]
Is it possible to customize the tree view of a colleciton?
Right now only the `title` field is shown. It would be great to show some additional custom fields. Is there a way to achieve this?

**Comment by jasonvarga** [comment]
You can show slugs by enabling it the collection settings.

![image](https://user-images.githubusercontent.com/105211/137918306-03e523ec-c9dc-4548-b0bb-0a8d76cbd1c0.png)

![image](https://user-images.githubusercontent.com/105211/137918360-125bdf2d-3967-42b3-b08e-9148cf6cec99.png)

**Reply by carstenjaksch** [reply]
@jasonvarga Are there more options today? Maybe via YAML configuration files or service provider.

**Reply by edalzell** [reply]
Nope
